packages/visualizations/src/components/Format/utils.ts

```diff
@@ -1,10 +1,19 @@
-export function isValidRawValue(rawValue: unknown): boolean {
-    return rawValue !== undefined && rawValue !== null;
+export function isValidValue(value: unknown): boolean {
+    return value !== undefined && value !== null;
 }
 
 export function warn(value: unknown, format: string) {
-    // eslint-disable-next-line no-console
-    console.warn(`ODS Dataviz SDK - Table: ${value} is not a valid ${format}`);
+    switch (format) {
```

Comment:
Usually I tend to add the "if (debugWarning)" in the debug function itself, so that future devs that use it don't have to remember to do the check themselves. Also it prevents having to pass `debugWarning` deep down. But I'm not sure that the store that holds the value is accessible here :/ My Svelte is rusty

Reply:
We have to subscribe manually, but `debugWarnings` store could be used in utils.ts indeed. (it changes in svelte 5 with .svelte.ts files).